neuromation/cli/job.py

```diff
@@ -889,6 +927,13 @@ def format_fail(job: str, reason: Exception) -> str:
     if not wait_start:
         detach = True
 
+    job_life_span = await calc_life_span(root.client, life_span)
+    if not root.quiet:
```

Comment:
I rather prefer to see this printout in explicit verbose mode only. In both normal and quiet modes the message should be absent.

Reply:
Isn't the information that your job will die automatically after certain (in some cases, implicit) amount of time a more important information rather than, say, Shortcuts? I think, if the job has a timeout, it should be printed in non-quiet mode. Other question is, perhaps, a better place for it is `formatters/jobs.py`?

Reply:
There is a lot of important information about the job. We need a balance between insufficient and too verbose outputs.
Moving reports about remaining time is a very cool idea.
I see the REST API inconsistency here though: the deadline timestamp is not returned in `JobDescription` data structure. Seems like the server doesn't file this info, that's sad.  In my mind, it should be an absolute UNIX timestamp; relative times are not reliable. The error is one less significant digit due to network latency, 1 minute in your case.
It raises another question: should the absolute timestamp be used as input in REST API?  Passing a relative time in Python API is ok.